src/net.h

```diff
@@ -1002,6 +1004,44 @@ class CConnman
     mapMsgTypeSize m_msgtype_bytes_recv;
     mapMsgTypeSize m_msgtype_bytes_sent;
 
+    struct NetMsgStatsKey {
+        // protocol.cpp has a list of allNetMessageTypes which use the NetMsgType namespace.
+        // Can I treat NetMsgType like an enum?
+        std::string msg_type;
+        ConnectionType conn_type;
+        Network net_type;
+
+        // Is this correct? Is there a better way to write this?
+        bool operator< (const NetMsgStatsKey& rhs) const{
```

Comment:
`grep -r -i --color=always "operator<("`

Take a look at some other implementations of this overloaded operator